doc/source/tutorial/signal.rst

```diff
@@ -905,86 +905,453 @@ in the amplitude response.
+:func:`~scipy.signal.windows.boxcar` window corresponds to a :math:`\rect` function,
+i.e., to no windowing. Furthermore, many of the depicted windows are more frequently
+used in filter design than in spectral analysis.
+
+
+
+Phase of Spectrum
+^^^^^^^^^^^^^^^^^
+The phase (i.e., :func:`~numpy.angle()`) of the Fourier transform is typically utilized
+for investigating the time delay of the spectral components of a signal passing through
+a system like a filter. In the following example the standard test signal, an impulse
+with unit power, is passed through a simple filter, which delays the input by three
+samples. The input consists of :math:`n=50` samples with sampling interval :math:`T =
+1` s. The plot shows magnitude and phase over frequency of the input and the output
+signal:
+
+.. plot:: tutorial/examples/signal_SpectralAnalysis_SpectrumPhaseDelay.py
+
+The input has a unit magnitude and zero-phase Fourier transform, which is the reason
+for the use as a test signal. The output has also unit magnitude but a linearly falling
+phase with a slope of :math:`-6\pi`. This is expected, since delaying a signal
+:math:`x(t)` by :math:`\Delta t` produces an additional linear phase term in its
+Fourier transform, i.e.,
+
+.. math::
+
+        \int_\IR x(t-\Delta t) \e^{-\jj2\pi f t}\dd t =
+        X(f)\,  e^{-\jj2\pi \Delta t f}\ .
+
+Note that in the plot the phase is not limited to the interval :math:`(+\pi, \pi]`
+(output of :func:`~numpy.angle`) and hence does not have any discontinuities. This is
+achieved by utilizing the :func:`numpy.unwrap` function. If the transfer function of
+the filter is known, :func:`~scipy.signal.freqz` can be used to determine the spectral
+response of a filter directly.
+
+
+Spectra with Averaging
+^^^^^^^^^^^^^^^^^^^^^^
+The :func:`~scipy.signal.periodogram` function calculates a power spectral density
+(``scaling='density'``) or a squared magnitude spectrum (``scaling='spectrum'``). To
+obtain a smoothed periodogram, the :func:`~scipy.signal.welch` function can be used. It
+does the smoothing by dividing the input signal into overlapping segments, to then
+calculate the windowed DFT of each segment. The result is to the average of thoseDFTs.
+
+.. Consult ``TestSampledSpectralRepresentations.test_windowed_DFT`` in file
+   `signal/test/test_spectral.py` for plausibility tests that the scalings of
+   ``periodogram`` and ``welch`` are consistent.
+
+The example below shows the squared magnitude spectrum and the power spectral density
+of a signal made up of a :math:`1.27\,\text{kHz}` sine signal with amplitude
+:math:`\sqrt{2}\,\text{V}` and additive gaussian noise having a spectral power density
+with mean of :math:`10^{-3}\,\text{V}^2/\text{Hz}`.
+
+.. plot:: tutorial/examples/signal_SpectralAnalysis_PeriodogramWelch.py
+
+The plots shows that the :func:`~scipy.signal.welch` function produces a much smoother
+noise floor at the expense of the frequency resolution. Due to the smoothing the height
```

Comment:
It is actually a bit hard to see the "expense of the frequency resolution" part because both peaks are quite narrow (even though the Welch one is slightly less narrow):

![image](https://github.com/scipy/scipy/assets/2365790/4bf35108-943d-4acb-a37e-20132f12b8fe)


Maybe zoom in from 1-2kHz instead of showing 0-5kHz?

Reply:
Good point, fixed in commit https://github.com/scipy/scipy/commit/27ad44bd7bfafb0e50004de41b6836a5dbdbdf27.